extraction/data_models/contents.py

```diff
@@ -0,0 +1,60 @@
+"""This script creates object structure to store the extracted data
+    and provides methods to set and retrieve the attributes"""
+
+import enum
+
+
+class Contents:
+    """This class creates content_list which is list of extracted contents"""
+    def __init__(self):
+        self.content_list = list()
+
+    def add_content(self, _content):
+        """This method appends extracted content to content_list"""
+        self.content_list.append(_content)
+
+    def __str__(self):
+        return_string = '[\n'
+
+        for content in self.content_list:
+            return_string = '{}{}\n\t}},\n'.format(return_string, str(content))
+
+        return return_string + ']'
+
+
+class _Content:
+    """This class creates object representation for each extracted content"""
+
+    counter = 0
+
+    def __init__(self, content_type):
+        _Content.counter += 1
+        self.index = _Content.counter
+        self.content_type = content_type
+
+    def __str__(self):
+        return '\t{{Index: {index}\n\t\t' \
+               'ContentType: {content_type}\n\t\t'.\
+            format(index=self.index, content_type=self.content_type.name)
+
+    def get_index(self):
+        """Returns the index of the content"""
+        return self.index
+
+    def get_content_type(self):
+        """Returns the content type of the content"""
+        return self.content_type
+
+
+class ContentType(enum.Enum):
+    """This class creates enumerated constants for Content Type"""
+
+    UNKNOWN = 1
+    TEXT = 2
+    IMAGE = 3
+    VIDEO = 4
+    QUOTE = 5
+    EMBEDDED_TWEET = 6
+    EMBEDDED_INSTAGRAM_POST = 7
+    EMBEDDED_PINTEREST_PIN = 8
+    EMBEDDED_YOUTUBE_VIDEO = 9
```

Comment:
Can we use the class name of the object for object type? We wouldn't need this enum then 

Since there's a 1-1 mapping from enum to class, maybe we don't need this enum

Reply:
I see, you are right. Though, I was thinking to keep it for now and if we don't use it for any other purpose we can remove it in next iterations.